test_ugp_decorator_shared unit test failure

```
2022-07-26T16:32:08.5446407Z ERROR [5.013s]: test_ugp_decorator_shared (tests.test_ugp.UgpTestCase)
2022-07-26T16:32:08.5449395Z ----------------------------------------------------------------------
2022-07-26T16:32:08.5452143Z Traceback (most recent call last):
2022-07-26T16:32:08.5455010Z   File "/python/deephaven/ugp.py", line 43, in exclusive_lock
2022-07-26T16:32:08.5457332Z     yield
2022-07-26T16:32:08.5463491Z   File "/python/tests/testbase.py", line 55, in wait_ticking_table_update
2022-07-26T16:32:08.5466363Z     self.assertGreaterEqual(table.size, row_count)
2022-07-26T16:32:08.5469167Z   File "/usr/lib/python3.7/unittest/case.py", line 1244, in assertGreaterEqual
2022-07-26T16:32:08.5473168Z     self.fail(self._formatMessage(msg, standardMsg))
2022-07-26T16:32:08.5476180Z   File "/usr/lib/python3.7/unittest/case.py", line 680, in fail
2022-07-26T16:32:08.5478872Z     raise self.failureException(msg)
2022-07-26T16:32:08.5481597Z AssertionError: 4 not greater than or equal to 5
2022-07-26T16:32:08.5482999Z 
2022-07-26T16:32:08.5484411Z The above exception was the direct cause of the following exception:
2022-07-26T16:32:08.5491754Z 
2022-07-26T16:32:08.5493627Z Traceback (most recent call last):
2022-07-26T16:32:08.5496495Z   File "/python/tests/test_ugp.py", line 85, in test_ugp_decorator_shared
2022-07-26T16:32:08.5499379Z     self.wait_ticking_table_update(t, row_count=5, timeout=5)
2022-07-26T16:32:08.5502285Z   File "/python/tests/testbase.py", line 55, in wait_ticking_table_update
2022-07-26T16:32:08.5505101Z     self.assertGreaterEqual(table.size, row_count)
2022-07-26T16:32:08.5508021Z   File "/usr/lib/python3.7/contextlib.py", line 130, in __exit__
2022-07-26T16:32:08.5510685Z     self.gen.throw(type, value, traceback)
2022-07-26T16:32:08.5513337Z   File "/python/deephaven/ugp.py", line 45, in exclusive_lock
2022-07-26T16:32:08.5516099Z     raise DHError(e, "exception raised in the enclosed code block.") from e
2022-07-26T16:32:08.5519131Z deephaven.dherror.DHError: exception raised in the enclosed code block. : AssertionError: 4 not greater than or equal to 5
2022-07-26T16:32:08.5521950Z Traceback (most recent call last):
2022-07-26T16:32:08.5524535Z   File "/python/deephaven/ugp.py", line 43, in exclusive_lock
2022-07-26T16:32:08.5527001Z     yield
2022-07-26T16:32:08.5529707Z   File "/python/tests/testbase.py", line 55, in wait_ticking_table_update
2022-07-26T16:32:08.5532588Z     self.assertGreaterEqual(table.size, row_count)
2022-07-26T16:32:08.5535686Z   File "/usr/lib/python3.7/unittest/case.py", line 1244, in assertGreaterEqual
2022-07-26T16:32:08.5538525Z     self.fail(self._formatMessage(msg, standardMsg))
2022-07-26T16:32:08.5541365Z   File "/usr/lib/python3.7/unittest/case.py", line 680, in fail
2022-07-26T16:32:08.5544063Z     raise self.failureException(msg)
2022-07-26T16:32:08.5546633Z AssertionError: 4 not greater than or equal to 5
```

--- a/py/server/tests/test_ugp.py
+++ b/py/server/tests/test_ugp.py
@@ -66,7 +66,7 @@ class UgpTestCase(BaseTestCase):
             t = ticking_table_op_decorated()
 
         t = ticking_table_op_decorated(5)
-        self.wait_ticking_table_update(t, row_count=5, timeout=5)
+        self.wait_ticking_table_update(t, row_count=5, timeout=10)
 
         t = ticking_table_op_decorated(10, "00:00:00.001")
         self.wait_ticking_table_update(t, row_count=8, timeout=5)
@@ -82,7 +82,7 @@ class UgpTestCase(BaseTestCase):
             t = ticking_table_op_decorated()
 
         t = ticking_table_op_decorated(5)
-        self.wait_ticking_table_update(t, row_count=5, timeout=5)
+        self.wait_ticking_table_update(t, row_count=5, timeout=10)
 
         t = ticking_table_op_decorated(10, "00:00:00.001")
         self.wait_ticking_table_update(t, row_count=8, timeout=5)

